arxiv_latex_cleaner/arxiv_latex_cleaner.py

```diff
@@ -175,8 +175,11 @@ def _remove_comments_inline(text):
 def _strip_tex_contents(lines, end_str):
   """Removes everything after end_str."""
   for i in range(len(lines)):
-    if end_str in lines[i]:
+    if end_str in lines[i] and '%' not in lines[i]:
       return lines[:i + 1]
+    if end_str in lines[i] and '%' in lines[i]:
+      if lines[i].index('%') > lines[i].index(end_str):
+        return lines[:i + 1]
```

Comment:
Can we simplify this bit as follows?

```
if end_str in lines[i]:
  if '%' not in lines[i]:
    return lines[:i + 1]
  elif lines[i].index('%') > lines[i].index(end_str):
    return lines[:i + 1]
```

Reply:
Hello @jponttuset,
Thank you and I have updated the code.